src/arguments.js

```diff
@@ -135,6 +135,21 @@ const removeExtraEnumValues = (enumArray, regexp) => {
     return result.length !== 0 ? result : null;
 };
 
+/**
+ * Parse argument property isFlag by examples section
+ * @param {string} section - examples section
+ * @param {object} context - internal config
+ */
+const parseIsFlag = (section, context) => {
```

Comment:
We need better name for this function

Reply:
@svoboda-rabstvo yes, maybe you have any ideas?